### ISSUE TITLE
[Bug]: Alpha-Modifiers are not transformed

### ISSUE THREAD

**Issue by niggeulimann** [tokens-studio/sd-transforms#164]
### What happened?

Using a alpha-modifier has no effect.

### Reproduction

https://configurator.tokens.studio/#project=jVRNj5swEP0rFl0JiAisKlWqUqXqYdX2WCl7K9HKAZO4CzayzTZRxH+vxzYEHCotB2Rm3ny8N4yvQcFZRY/pH8lZsAmuOUMoDyTvREHyYIN+gwFMq1W2ShV/JUwacB6AZ5/YgLbGquKikRBzHWIwKwWn5cQGViUwkwD+IXjXgjMPik4q3mQ2/1qqrqQ8D5JbUCtIRc8WLMuZ69DRuvyF1cl6eafaTmUzSEVrIid04LnejgApiVSUYUWBmmmJ11yk56aeZHLZdO9YWZSjmAliNZN36MHzfGnJJLPVzz79cNzbg/nWrz5IdPiRSkXE7ul7xwpoD/TXo6JNy4VCO3WpyRM1HiwuqBK8QaEE67oczeGXnLmAKxpSPg+DkAkahyJR73J8m00jk+X6Bprk06pdDCMXNn5/BFDO7qtFXs+xwXnG9C4uMhNTWkYQ8Q3XHXFiM9yQ6TCcwMZneqaKvmmEEh0xRj2+4kTEBkWGY4y2X5E5pZAebbdbFJok4S0HtOCHDFStLTU9xXpLfpLz550SlB2jWBPR21GQKKuqh+yYoDCMddb+nazNkljqlma4tCten1L/6WmaqsmAQ4jOdJ2s0Ic61JaZXOHeNqV/OZtyuuy3m8Fg51t+0OxevR1304GhiOMBRx8ezZOgx/RTPFtNtbwWfQLqjCsOJV7+vjS8pBUlYlbNq+dKzfZwoczM+0DOSnOF5fJSm9twqsed30BMY5cln1cd1+0Je3fEAgkt039AssWFA0kt7fQa8S4T72M8OoHReMn0/wA=

### Expected output

Both colors should have the value #00000080

### Version

0.9.3

**Comment by jorenbroekema** [comment]
Fixed [see example](https://configurator.tokens.studio/#project=jVTNjpswEH4Vi64ERARWlSpVqVL1sGp7rJS9lWjlgEncBRvZZpsI8e712CYYklblgIaZb/6+YaYPCs4qekx/Sc6CTdDnDKE8kLwTBcmDDfoJClCtVtkqVfyVMGnAeQCWfWId2hqriotGgk8/+mBWCk5LTwdaJTCTAP4meNeCMQ+KTireZDb+WqqupDwPksmpFaSiZwuW5cx06Ghd/sDqZK28U22nshmkojWRXjvw9JMIkJJIRRlWFFozJfGai/Tc1F4kF03XjpVFuRYzQSxn8gY9Wp4vLfEiW/7sM4zi3grmW7+GINHuRyoVEbunrx0roDzgX4+KNi0XCu3UpSZP1FiwuKBK8AaFErTr8qoOP+XMOfRoDPk8DkIm6DoUiQYX48tsGpks1xPIi6dZu5iOnNv1+z2AcnabLVrUHBvcQpne+EVmYkrTCCS+4bojjmyGG+IPwxFsbKZmquibRijREaPU4ytORGxQZHqM0fYzMlIK4dF2u0WhCRJOMaCEpcvYqtWlpqZYb8l3cv64U4KyYxTrRvR2FCTKquohOyYoDGMddfjPrs2S2NZtm+G9XVnUKfWfnqap8gYcgnem82SFFupQa2Z0hXtblP7lbEh/2afLYLDzLT/o7l4XO+6mA0MRxwOO3j2aJ0GP6Yd4tprq/loMCbDjJXj5/dLwklaUiL/ncmn+ncCzPZCz0j3CUs2Cmhvos7Cwgt1Uc7kxzFPiuj3hxUlY1KwZuQOQLS4cQGoG7yD8K3QiZ/+eeBfFE50woOt1Gf4A)

Basically, it's the combination of doing the modification in srgb space, it then outputs to rgba() format, and then your own tinycolor2 transform seems to mess up the 8 digit hex alpha value somehow. You can quite easily specify the output format in the color modification as an extra prop, and since today you can also pass the output format for color modifications on a global level, [see readme](https://github.com/tokens-studio/sd-transforms#options)
